.env.example

```diff
@@ -35,6 +35,9 @@ CORE_JSON_RPC_PASSWORD=password
 ISOLATED_ST_UNSERIALIZATION_MEMORY_LIMIT=10
 ISOLATED_ST_UNSERIALIZATION_TIMEOUT_MILLIS=300
 
+# requests
+MAX_IDENTITIES_PER_REQUEST=25
```

Comment:
Do you think we should configure it? We don't configure the limit for documents. It's hardcoded and supposed to be the same on all nodes. I guess it's the same case.